Makefile

```diff
@@ -0,0 +1,4 @@
+clean:
+	rm -rf work/
+	rm -rf .nextflow.log*
+	cd tests && rm -rf work && rm -rf .nextflow.log* && rm -rf .nextflow/
```

Comment:
Cool! I like the idea of having a make file instead of a test script to handle this stuff.

Maybe we could even build it into the [nf-core helper package](https://github.com/nf-core/tools).. 🤔